types/node-easy-cert/tslint.json

```diff
@@ -0,0 +1,4 @@
+{
+    "extends": "dtslint/dt.json",
+    "strict-export-declare-modifiers": true
```

Comment:
I believe this is true by default. You can remove this line.

Reply:
Fixed in last commit